### PR TITLE
git actions: Add PR template

### PR DESCRIPTION
**Pull request by oshoval**
Signed-off-by: Or Shoval <oshoval@redhat.com>

```release-note
None
```

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Before you click the 'Create pull request' make sure that:
+- This PR introduces a single feature of fix, just one
+- This PR does not leave the main branch broken
+- Every commit in this PR has a commit message explaining what do you change,
+  why and what is the outcome
+- If your change introduces a complex concept or a change to user interaction
+  with the project or the application, make sure to document it
+
+If you don't comply with these rules, you waste your energy, time of reviewers
+and cause suffering of future generations.
```
Comment (AlonaKaplan) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: I find this sentence redundant.

**Comment by oshoval** [comment]
Addressed comments

**Comment by AlonaKaplan** [comment]
@oshoval  Side note, please make sure the release note are actually collected and added to the release.

**Comment by oshoval** [comment]
> @oshoval Side note, please make sure the release note are actually collected and added to the release.

If there is no automation for it, i will remove it for now, because i can't create it at the near future

**Comment by AlonaKaplan** [comment]
> > @oshoval Side note, please make sure the release note are actually collected and added to the release.
> 
> If there is no automation for it, i can remove it for now, because i can't create automation at the near future unless we have examples to be based on

Let's start from checking it is is magically works. If not, let's remove it from the template and from prow.

**Comment by AlonaKaplan** [comment]
/approve cancel
let's first verify the release notes are working.

**Comment by oshoval** [comment]
Rebase

**Comment by oshoval** [comment]
It seems we need to add something like this
https://github.com/nmstate/kubernetes-nmstate/blob/main/hack/render-release-notes.sh#L14

CNAO doesn't collect, k8s nmstate does
I prefer to not dive into it at the moment, first to close other stuff, so i can remove it from the template / plugin 
but imo it is fine that we keep it there, and just improve later to collect it ?

**Pull request merged**